src/apps/sources/__test__/pages/CreateSourcePage.test.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import CreateSourcePage, {mapActionsToProps, mapStateToProps} from "../../pages/CreateSourcePage";
+import {APIOrg, APIProfile} from "../../../authentication";
+import {APISource, Source} from "../../types";
+import {createSourceAction, createSourceDispatchAction} from "../../redux";
+import {act, fireEvent, render} from "@testing-library/react";
+import {Provider} from "react-redux";
+import store from "../../../../redux";
+import {BrowserRouter as Router} from "react-router-dom";
+import {ViewSourcePage} from "../../pages";
+import {CONTEXT} from "../../constants";
+import {currentState, personalSources, testSource} from "../test_data";
```

Comment:
@binduak @supriyatw can we remove all the unused imports

Reply:
@rakesh-1010 Done !